datawig/simple_imputer.py

```diff
@@ -184,7 +183,7 @@ def fit_hpo(self,
         """
 
         if weight_decay is None:
-            weight_decay = [0]
+            weight_decay = [0, 1e-4, 1e-3, 1e-2]
```

Comment:
are these parameter candidates sensible? i mean have you seen improvements in the majority of experiments with any of those? if now, they'll merely multiply your training time by 4 which can be significant.

Reply:
The fit_hpo method should actually be removed. It's replaced by the new module. So no need to discuss this.

Reply:
hm. not sure whether fit_hpo should be removed. but my point was about the parameter choices. have you seen any improvement with any of those values in experiments?